CMakeLists.txt

```diff
@@ -118,7 +121,14 @@ set_target_properties(RegsGen2::RegsGen2 PROPERTIES
   IMPORTED_LOCATION ${BINARY_DIR}/${RegsGen2_RELATIVE_PATH})
 add_dependencies(RegsGen2::RegsGen2 RegsGen2)
 
-foreach(ARCH ARM ARM64 X86 X86_64)
+# Generate different pointer size RISC-V definitions.
+# FIXME(compnerd) this assumes all targets are RISCVnnnG, which may not be true.
+foreach(WORDSIZE 32 64 128)
+  configure_file(${PROJECT_SOURCE_DIR}/Definitions/RISCVnnn.json
```

Comment:
Nit: More clear name than RISCVnnn? Isn't exactly clear what it represents.

Reply:
I took this name from one of the RISCV docs (ELF psABI?).  The "nnn" is a placeholder for digits indicating the bitness.